packages/libchamplain.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Libchamplain < Package
+class Libchamplain < Meson
   description 'A map widget'
   homepage 'https://wiki.gnome.org/Projects/libchamplain'
   version '0.12.20'
```

Comment:
Update to [0.12.21](https://gitlab.gnome.org/GNOME/libchamplain/-/tags/0.12.21)?